geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa-accumulo

```diff
@@ -44,6 +44,8 @@ if [[ -n "${GEOMESA_EXTRA_CLASSPATHS}" ]]; then
 fi
 
 ACCUMULO_CP="$(findJars $ACCUMULO_LIB true true)"
+# if there's a geomesa runtime jar in accumulo, exclude it from the classpath
+ACCUMULO_CP="${ACCUMULO_CP#geomesa-accumulo-distributed-runtime*jar}"
```

Comment:
Wow.  Next, these scripts are gonna start holding people's hands...

Reply:
oops, this shouldn't be in here... it doesn't work